packages/lesswrong/server/scripts/backfillParentTags.ts

```diff
@@ -0,0 +1,45 @@
+import TagRels from "../../lib/collections/tagRels/collection";
+import Tags from "../../lib/collections/tags/collection";
+import { Globals } from "../vulcan-lib";
+import { v4 as uuid } from 'uuid';
+import { updatePostDenormalizedTags } from "../tagging/tagCallbacks";
+
+const backfillParentTags = async (parentTagSlug: string) => {
+  const parentTag = (await Tags.find({slug: parentTagSlug}).fetch())[0];
+  const childTags = (await Tags.find({parentTagId: parentTag._id}).fetch());
+  for(const childTag of childTags) {
+    const parentTagRelPostIds = (await TagRels.find({tagId: parentTag._id}).fetch()).map(rel => rel.postId);
+  
+    const childTagRelPostIds = (await TagRels.find({tagId: childTag._id}).fetch()).filter(rel => !parentTagRelPostIds.includes(rel.postId)).map(rel => rel.postId);
+  
+    const parentTagRelIds = childTagRelPostIds.map(_ => uuid({}).slice(0, 17));
+  
+    await TagRels.rawCollection().bulkWrite(childTagRelPostIds.map((postId, i) => ({
```

Comment:
I want to propose using performVoteServer to add the tag. We can have an admin account do the voting. I think that will leave the DB in a better state, and also reduce the possibility of something going wrong with the manual insertion.

You might be able to talk me out of this.

Reply:
Ah, of course, I forgot about that constraint. Nevermind, :shipit: